### DISCUSSION TITLE
Notebook: 15 Peclet interface-coupled 1D with Dirichlet Boundary Conditions.

### DISCUSSION THREAD

**Discussion by AnthonyB08** [dpploy/engy-5310#65] [Q&A] [answered]
In the [normal-flux-continuity] interface kernel, is type  = "InterfaceNormalFluxContinuity" a MOOSE object or user? It is returning an error of " A  InterfaceNormalFluxContinuity is not a registered object".  On their website it is InterfaceDiffusionFluxMatch, but even then I am getting the same error.

**Comment by dealmeidavf** [comment, marked as answer]
[InterfaceNormalFluxContinuity](https://github.com/dpploy/engy-5310/blob/main/moose-app-templates/InterfaceNormalFluxContinuity.h) is a derived class from `InterfaceKernel` as shown in the template. Make sure to change the `registerMooseObj` in the `.C` corresponding file of the course template.
